src/components/DatePicker/DatePickerPopover.jsx

```diff
@@ -94,7 +94,7 @@ DatePickerPopover.propTypes = {
 const DefaultInput = forwardRef(({ className, ...rest }, reference) => {
     return (
         <div ref={reference} className="relative flex bg-gray-lighter">
-            <div className="pointer-events-none absolute inset-0 flex items-center pl-3">
+            <div className="pointer-events-none absolute inset-0 z-10 flex items-center pl-3">
```

Comment:
Not sure if this is right, inputs should not have a default z-index. It should depend on where it's used

Reply:
I move the z-index to the icon. It's happening because the parent div has `absolute`

Reply:
How do you think we should solve it if we don't want to put the `z-10` on the icon